plugins/linux-capture/xcomposite-input.c

```diff
@@ -0,0 +1,400 @@
+#include <obs-module.h>
+#include <glad/glad.h>
+#include <glad/glad_glx.h>
+#include <X11/Xlib-xcb.h>
+#include <xcb/xcb.h>
+#include <xcb/composite.h>
+#include <pthread.h>
+
+#include <stdlib.h>
+#include <string.h>
+#include <stdio.h>
+#include <ctype.h>
+
+#include "xhelpers.h"
+#include "xcursor-xcb.h"
+#include "xcomposite-input.h"
+#include "xcomposite-helpers.h"
+
+static Display *disp = NULL;
+static xcb_connection_t *conn = NULL;
+
+static void xcompcap_update(void *data, obs_data_t *settings);
+
+static uint32_t xcompcap_getwidth(void *data)
```

Comment:
Since you're already touching this code anyway, would you mind adding an underscore to `get_width` (same for `get_height`)? More generally, I think it would be good to make the suffix of these static functions match the vfuncs of `struct obs_source_info`

Reply:
Done, good idea